spec/system/provider_interface/provider_applications_pagination_spec.rb

```diff
@@ -58,15 +58,20 @@ def then_i_should_not_see_a_paginator
     expect(page).not_to have_content('Showing 1 to')
   end
 
+  # rubocop:disable RSpec/FactoryBot/CreateList
```

Comment:
I think this is the only controversial change. Rather than `30.times`, Rubocop wants us to use `create_list`. However, that will only call `course_option_for_provider` once (ie it will set the same course option on each application choice), resulting in validation errors. What _should_ work is `create_list(blah) { |choice, _| ... }` with a block that sets the `course_option` property, but it made the test fail. Hence I've resorted to `#rubocop:disable` and there is no longer a place reserved for me in Ruby Heaven.